app/assets/javascripts/quby/answers/validation.js

```diff
@@ -221,20 +227,45 @@
     var answered = 0;
     var visible = 0;
     var hidden = 0;
+    var total = 0;
+    var countedKeys = [];
+    var answeredKeys = [];
 
     for(var i = 0; i < groupItems.length; i++){
-      var inputs = $(groupItems[i]).find("input, textarea, select").not(":disabled, :hidden");
+      var item = $(groupItems[i]);
+      var inputs = item.find("input, textarea, select").not(":disabled, :hidden");
+      var countsFor = 1;
+      var answersFor = 1;
+
+      if(item.is('.option')){
```

Comment:
.options met groups komen alleen in tabellen voor, in tabellen hebben deze altijd een data for.

De countsfor oplossing is zodat de mix van items en options die in tabellen voor komt goed samen werken. Items buiten tabellen geven hun question key weer op een andere manier door, dus dat is ook weer meer code.

Ja dit is allemaal heel lelijk.